### PR TITLE
Lesson 2

### PR DESCRIPTION
**Pull request by RandyCastr0**
completed quiz

### CODE REVIEW

File: lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ private static QuizQuestion makeQuestion0() {
             AnswerChoice.B, "The monitor",
             AnswerChoice.C, "The keyboard",
             AnswerChoice.D, "The mouse"),
-        AnswerChoice.UNANSWERED); // Replace `UNANSWERED` with the correct answer.
+        AnswerChoice. A); // Replace `UNANSWERED` with the correct answer.
```
Comment (anthonydmays) [inline lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java:55]: Need to fix formatting here and below. Run `./gradlew spotlessApply" and then push your changes per the updated instructions.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java [hunk 2]
```diff
@@ -112,21 +112,21 @@ private static QuizQuestion makeQuestion5() {
...
-        AnswerChoice.UNANSWERED); // Replace `UNANSWERED` with the correct answer.
+        AnswerChoice.A); // Replace `UNANSWERED` with the correct answer.
   }
 
   private static QuizQuestion makeQuestion6() {
     return new QuizQuestion(
         6,
         "What is the decimal number 34 in binary?",
-        "0b00000000"); // Replace the correct digits in the answer.
+        "0b00100010."); // Replace the correct digits in the answer.
```
Comment (anthonydmays) [inline lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java:122]: Extra period in answer, please fix.

File: lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java [hunk 3]
```diff
@@ -112,21 +112,21 @@ private static QuizQuestion makeQuestion5() {
...
   }
 
   private static QuizQuestion makeQuestion7() {
     return new QuizQuestion(
         7,
         "What is the decimal number 34 in hex?",
-        "0x00"); // Replace the correct digits in the answer.
+        "22"); // Replace the correct digits in the answer.
```
Comment (anthonydmays) [inline lesson_02/quiz/lesson_02_quiz/src/main/java/com/codedifferently/lesson2/Lesson2.java:129]: Answer not provided correctly, the "0x" part matters.

**Review by anthonydmays — CHANGES_REQUESTED** [review]
Five of your answers are incorrect. Keep working on it.

**Comment by RandyCastr0** [comment]
fixed answers, I think it didn't merge right the first time because I had everything correct and ran/formatted  right the first time.

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request closed without merging**